src/app/Game.ts

```diff
@@ -16,6 +18,30 @@ interface TargetingState {
 
 export namespace Game {
```

Comment:
nether of these need to use `export namespace`, you can just use es6 modules. nesting namespaces like this is also not great; if the debug stuff needs to be separate it should be in a separate module

Reply:
Debug has been moved to its own namespace. As for namespace VS modules, I'm not opposed to switching from namespaces to modules, but I personally would rather focus my energies elsewhere.

Reply:
thats fine, although  I'm pretty sure you're overestimating how much effort it would be to change this